src/main/scala/org/ergoplatform/nodeView/state/ErgoStateContext.scala

```diff
@@ -0,0 +1,58 @@
+package org.ergoplatform.nodeView.state
+
+import com.google.common.primitives.{Bytes, Shorts}
+import org.ergoplatform.modifiers.history.{Header, HeaderSerializer}
+import org.ergoplatform.settings.Constants
+import scorex.core.serialization.{BytesSerializable, Serializer}
+import scorex.crypto.authds.ADDigest
+
+import scala.util.Try
+
+/**
+  * Additional data, required for transactions validation.
```

Comment:
Additional data required for transactions validation

Reply:
fixed